extension/manifest.json

```diff
@@ -16,14 +16,21 @@
 			"js": [
 				"js/copy.js",
 				"js/warning.js",
-				"js/colour.js"
+				"js/colour.js",
+				"js/pdf.js"
 			]
 		}
 	],
 	"content_security_policy": {},
 	"host_permissions": [
-		"https://*.librarything.com/*"
+		"https://*.librarything.com/*",
+		"https://*.b-ok.cc/*",
+		"https://*.libgen.is/*"
```

Comment:
why do we need to do this?

Reply:
This allows the plugin network access to these websites... I think

Reply:
Just checked. It is required.